Jenkinsfile

```diff
@@ -272,6 +272,9 @@ def executeStandardTests() {
     . ${HOME}/environment.txt
     set +o allexport
 
+    . ${JABBA_SHELL}
```

Comment:
nit: should we extract this as a method and reuse it where needed?

Reply:
Perhaps, but there are some downsides to that approach.  Each of these shell blocks start up with zero context so you have to configure everything each time.  There are some config options in the openstack-jenkins-drivers repo which provides tools to help a bit with this; that's where environment.txt is built, for instance.  So we could add the function there and just call it here; that would be the easiest way to make it fairly common.

Problem with that is that your Jenkinsfile is now more explicitly tied to the configuration of the runner.  We already have numerous other cases of that in the build right now so it's not like you're creating a _new_ dependency there.  But if we were wanting to change that I'd want it to go in the other way; I'd rather have this build be more self-contained so that we can get as close as possible to the idea of taking this Jenkinsfile, dropping it in a new Jenkins install and largely having what we need.  We'll never have that completely; there's too much config built into openstack-jenkins-drivers as-is.  But I'd like to at least keep moving in that direction.